mesh/net.c

```diff
@@ -2619,13 +2621,16 @@ static void process_beacon(void *net_ptr, void *user_data)
 	 */
 	if (net->iv_upd_state == IV_UPD_INIT || ivi != net->iv_index ||
 							ivu != net->iv_update)
-		update_iv_ivu_state(net, ivi, ivu);
+		updated = update_iv_ivu_state(net, ivi, ivu);
 
 	if (kr != local_kr)
 		update_kr_state(subnet, kr, beacon_data->key_id);
```

Comment:
Why not here as well?

Reply:
I had to read about `Key Refresh` mechanism. I've fixed it.